oap-server/server-core/src/main/java/org/apache/skywalking/oap/server/core/query/NotGraphQLField.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package org.apache.skywalking.oap.server.core.query;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Target;
+
+/**
+ * NotGraphQLField represents the annotated field is not used by GraphQL protocol.
+ *
+ * Notice, this is a dev-level friendly label, there is no practical use in the runtime.
+ *
+ * @since 9.4.0
+ */
+@Target({ElementType.FIELD})
+public @interface NotGraphQLField {
```

Comment:
This annotation looks to be useless

Reply:
I just don't want comments everywhere. This annotation would not be compiled into the binary.

Reply:
> I just don't want comments everywhere. This annotation would not be compiled into the binary.

It WOULD be compiled into binary, if you want to skip compiling it into binary, please annotation with `@Retention(java.lang.annotation.RetentionPolicy.SOURCE)`, the default retention policy is runtime